run.py

```diff
@@ -43,9 +43,16 @@
 def get_sessions_pi_copy(fw_project: ProjectOutput) -> list:
     """Get and filter sessions for pi_copy()"""
     sessions = []
+    now = datetime.now(timezone.utc)
+    PST = timezone(timedelta(hours=-8))
     for session in fw_project.sessions():
         if any(tag.startswith('copied_') for tag in session.tags):
             continue
+        timestamp = session.timestamp.replace(tzinfo=PST).astimezone(timezone.utc)
+        if (now - timestamp) < timedelta(hours=6):
+            log.info('Skipping pi_copy of session %s because less than 6 hours have passed.' % session.label)
+            continue
```

Comment:
This is logic is duplicated now, best to split it off into a standalone helper function, perhaps something like:

```
def timedelta_from_now(in_timestamp: datetime, tz_from_utc: int = -8) -> timedelta:
```

Reply:
Just added this. I also included an update to create_view_df(), which probably should have been its own commit but oh well.